.projenrc.ts

```diff
@@ -1,19 +1,80 @@
 import { CdklabsConstructLibrary } from 'cdklabs-projen-project-types';
+import { DependencyType, JsonFile, JsonPatch } from 'projen';
+import { NpmAccess } from 'projen/lib/javascript';
 
 const project = new CdklabsConstructLibrary({
   author: 'AWS',
   authorAddress: 'aws-cdk-dev@amazon.com',
   cdkVersion: '2.80.0',
+  jsiiVersion: '~5.0.0',
   defaultReleaseBranch: 'main',
   name: '@aws-cdk/aws-lambda-dotnet',
   projenrcTs: true,
-  release: false,
+  npmAccess: NpmAccess.PUBLIC,
+  stability: 'experimental',
   repositoryUrl: 'https://github.com/cdklabs/awscdk-lambda-dotnet.git',
   autoApproveOptions: {
     allowedUsernames: ['aws-cdk-automation', 'dependabot[bot]', 'mergify[bot]'],
     secret: 'GITHUB_TOKEN',
   },
   autoApproveUpgrades: true,
+  prettier: true,
+  gitignore: [
+    '*.d.ts',
+    '*.generated.ts',
+    '*.js',
+    '**/integ.*.snapshot/asset.*/',
+    '**/bin',
+    '**/obj',
+    '/*.sln',
+  ],
+  publishToPypi: {
+    distName: 'cdklabs.aws-lambda-dotnet',
+    module: 'cdklabs.aws_lambda_dotnet',
+  },
+  publishToNuget: {
+    packageId: 'Cdklabs.AWS.Lambda.DotNet',
+    dotNetNamespace: 'Cdklabs.AWS.Lambda.DotNet',
+  },
+  publishToMaven: {
+    mavenGroupId: 'io.github.cdklabs',
+    javaPackage: 'io.github.cdklabs.aws.lambda.dotnet',
+    mavenArtifactId: 'cdklabs-aws-lambda-dotnet',
+    mavenEndpoint: 'https://s01.oss.sonatype.org',
+  },
+});
+
+project.package.addField('prettier', {
+  singleQuote: true,
+  semi: true,
+  trailingComma: 'es5',
 });
+project.eslint?.addRules({
+  'prettier/prettier': [
+    'error',
+    { singleQuote: true, semi: true, trailingComma: 'es5' },
+  ],
+});
```

Comment:
Use `prettierOptions` instead please. But also, except for `singleQuote` I don't think we want the other settings (`semi` is true anyway, `trailingComma` just adds them in a few more places when using the default `all` over `es5`)
```suggestion
```